tutorials/transfer-bucket-to-object-storage-hive/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+meta:
+  title: How to transfer Object Storage buckets to the new storage platform
+  description: This page explains how transfer exisiting Object Storage platforms to the new storage platform 
+content:
+  h1: How to transfer Object Storage buckets to the new storage platform
+  paragraph: This page explains how transfer exisiting Object Storage platforms to the new storage platform 
+tags: Object-Storage Storage Bucket S3 Hive
+categories:
+  - storage
+  - object-storage
+dates:
+  validation: 2022-01-28
+  posted: 2022-01-28
+---
+
+## Overview
+
+Since the 27th of January 2022 the [Object Storage](/storage/object/quickstart/) platform has been migrated to our internally developed backend: [Hive](https://papers.s3.fr-par.scw.cloud/hive.pdf).
+We use the robust Erasure Code (6+3) to protect the data stored with Object Storage. This mechanism now moves from one datacenter to three Availability Zones (AZ), which means that the 6+3 fragments will be spread over three independent Availability Zones in one region.
+In summary, the main advantage of Multi AZ is that it ensures the resilience of your data storage, even in the event of a total loss of an AZ due to accidents or disasters such as fire, flood, earthquake, power or network failures, etc.
+Our Object Storage offering is built entirely on Scaleway's technical stack, guaranteeing you true sovereignty at all levels, from infrastructure to software. Thanks to our new backend, you can store millions of objects in a single bucket and scale your infrastructure, without friction and without limit. 
+
+You can easily transfer the contents of your existing buckets to newly created buckets using the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli) to profit from the improvements of the new platform.
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [configured your API keys](/console/my-project/how-to/generate-api-key/)
+  - You have installed the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli) on your local computer
+
+</Message>
+
+## Data transfer using the AWS CLI 
+
+1. [Create](https://console.scaleway.com/object-storage/buckets/create) a new bucket in the Paris region by clicking on **Object Storage** in the **Storage** section of the Scaleway console. Then click **+ Create a bucket**.
+2. Run a full sync of the two buckets:
+    ```
+    aws s3 sync s3://BUCKET-SOURCE s3://BUCKET-TARGET
+    ```
+    <Message type="note">
+
+    Depending on the number of objects in your bucket, this step may take a while.
+
+    </Message>
+
+3. Run a second full sync to ensure that new objects, that have been added to the source bucket in between are also copied.
+4. Point your application to the new bucket.
+5. Recursive copy from the old bucket to the new bucket to ensure that new objects added in between are also copied:
```

Comment:
pas sur qu'on comprenne le "recursive copy"

Reply:
To copy recursively means that all files within a directory will be copied, and then all files within subdirectories of that directory, and so on.